test/Microsoft.AspNetCore.Blazor.Build.Test/RuntimeCodeGenerationTest.cs

```diff
@@ -1269,5 +1269,219 @@ public void Component_WithDocType()
             AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
             CompileToAssembly(generated);
         }
+
+        [Fact]
+        public void RazorTemplate_InCodeBlock()
+        {
+            // Arrange
+
+            // Act
+            var generated = CompileToCSharp(@"
+@{
+    RenderFragment<Person> p = @<div>@item.Name</div>;
+}
+@functions {
+    class Person
+    {
+        public string Name { get; set; }
+    }
+}");
+
+            // Assert
+            AssertDocumentNodeMatchesBaseline(generated.CodeDocument);
+            AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
+            CompileToAssembly(generated);
+        }
+
+
+        [Fact]
+        public void RazorTemplate_InExplicitExpression()
+        {
+            // Arrange
+
+            // Act
+            var generated = CompileToCSharp(@"
+@(RenderPerson(@<div>@item.Name</div>))
+@functions {
+    class Person
+    {
+        public string Name { get; set; }
+    }
+
+    object RenderPerson(RenderFragment<Person> p) => null;
+}");
+
+            // Assert
+            AssertDocumentNodeMatchesBaseline(generated.CodeDocument);
+            AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
+            CompileToAssembly(generated);
+        }
+
+        [Fact]
+        public void RazorTemplate_InImplicitExpression()
```

Comment:
The test cases for `InExplicitExpression` and `InImplicitExpression` appear to be identical. Am I missing something, or is this a copy-paste error?